src/interfaces/IDetectorOpts.ts

```diff
@@ -0,0 +1,24 @@
+enum DetectionConvertion {
+  RELATIVE = 'RELATIVE',
+  FRAMING = 'FRAMING',
+}
+/**
+ * Options used for detector
+ *
+ * @interface DetectorOpts
+ */
+export default interface DetectorOpts {
+  /**
+   * Type of bbox coordinates you when listening
```

Comment:
A bit confusing the explanation!

Reply:
Maybe you can say `Type of bbox coordinates when listening to DETECTIONS topic`!